docs/wallet-integration-guide/examples/scripts/exchange-integration/setup-customer.ts

```diff
@@ -2,18 +2,15 @@ import {
     WalletSDKImpl,
```

Comment:
Consider calling this `setup-demo-customer.ts`. Want to avoid any confusion that this is a required step.